qa/rpc-tests/test_framework/blockchainhelper.py

```diff
@@ -173,7 +173,7 @@ def create_sidechain(self, sc_name, sidechain_version, optional_parameters = {},
         and the version are mandatory.
         """
 
-        sc_input = SidechainCreationInput.from_rpc_args(self, sc_name, optional_parameters)
+        sc_input = SidechainCreationInput.from_rpc_args(self, sc_name, sidechain_version, optional_parameters)
         sc_input.version = sidechain_version
```

Comment:
This should no longer be needed, as `from_rpc_args` passes the value `sidechain_version` to `SidechainCreationInput` constructor. All in all, maybe it was possible to correctly set the SC version by passing a proper key-value in the `optional_parameters` dictionary

Reply:
I removed line 177 in f1aa8bb237c592621a4f635666ee5a4761bbb988. We may update to only use `optional_parameters` in future.